emails/models.py

```diff
@@ -36,6 +36,17 @@ class CannotMakeAddressException(Exception):
     pass
 
 
+'''
+class ReplyAddress(models.Model):
+    address = models.CharField(
+        max_length=64, default=address_default, unique=True
+    )
+    sender = models.CharField(
+        max_length=64, default=address_default, unique=True
```

Comment:
we should also add a `validators=[ensure_sha256_hash]` here. to make sure we never store plaintext values.

```python
def ensure_hash(value):
    if (not all(c in string.hexdigits for c in value) or 
        not len(value) == 64):
        raise ValidationError()
```